core/src/main/resources/org/apache/spark/ui/static/sorttable.js

```diff
@@ -3,19 +3,19 @@
   version 2
   7th April 2007
   Stuart Langridge, http://www.kryogenix.org/code/browser/sorttable/
-  
+
```

Comment:
It's not a big deal, but stripping whitespace on lines just adds to the PR noise and possibility of merge conflicts. If you can disable this it's much preferred.